recipes/qiskit-xyz2pdb/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "qiskit-xyz2pdb" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aabd4c37258d9d45db8c0cf89b19368497de6d7c15a514bb9f616b3f5871eab9
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - qiskit-xyz2pdb=xyz2pdb.xyz2pdb:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - qiskit-xyz2pdb
```

Comment:
This can not be the module name, isn't it?

Reply:
just updated to correct name :).